clippy_dev/src/main.rs

```diff
@@ -278,11 +279,19 @@ fn get_clap_config<'a>() -> ArgMatches<'a> {
                         Lint a package directory:
                             cargo dev lint tests/ui-cargo/wildcard_dependencies/fail
                             cargo dev lint ~/my-project
+
+                        Run rustfix:
+                            cargo dev lint ~/my-project -- --fix
```

Comment:
Would you mind adding an example for `--force-warn` or `-A`, I've tried these and both didn't work:

```
cargo dev lint tests/ui-cargo/wildcard_dependencies/fail -- -Aclippy::wildcard-dependencies
cargo dev lint tests/ui-cargo/wildcard_dependencies/fail -- -- -Aclippy::wildcard-dependencies
```

This could just be me and my missing knowledge of how rustc/cargo arguments correctly work :sweat_smile:

Reply:
huh 🤔

Reply:
Ahhh, that makes sense :sweat_smile: